grpc-interface-support/src/python.py

```diff
@@ -41,7 +44,49 @@ def get_required_sdk_version_python() -> str:
     return sdk_version
 
 
+def get_template_dir() -> str:
+    return os.path.join(
+        get_package_path(), "grpc-interface-support", "data", "templates", "python"
+    )
+
+
+class GrpcCodeExtractor:
+    """
+    Provides methods for extracting code from generated gRPC python files.
+    """
+
+    def __init__(
+        self, proto_file: ProtoFileHandle, base_path: str, source_path: str = "."
+    ):
+        self.__proto_file = proto_file
+        self.__base_path = base_path
+        self.__source_path = source_path
+
+        self.grpc_source_path = os.path.join(
+            self.__base_path,
+            self.__source_path,
+            f"{self.__proto_file.get_service_name().lower()}_pb2_grpc.py",
```

Comment:
```suggestion
            f"{Path(self.__proto_file.file_path).stem}_pb2_grpc.py",
```

Reply:
Extracted it to a variable for easy access in different functions